framework/src/test/java/viewpointtest/ViewpointTestQualifierHierarchy.java

```diff
@@ -0,0 +1,40 @@
+package viewpointtest;
+
+import org.checkerframework.framework.type.GenericAnnotatedTypeFactory;
+import org.checkerframework.framework.type.NoElementQualifierHierarchy;
+
+import java.lang.annotation.Annotation;
+import java.util.Collection;
+
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.util.Elements;
+
+import viewpointtest.quals.Bottom;
+import viewpointtest.quals.Lost;
+
+/** A qualifier hierarchy for the ViewpointTest checker. */
```

Comment:
```suggestion
/** A qualifier hierarchy for the Viewpoint Test Checker. */
```

Or a link to the class?

Reply:
Added link the QualifierHierarchy class.